test/sql/parser/star_expression.test

```diff
@@ -65,5 +65,3 @@ not supported
 
 statement error
 FROM range(*, *);
-----
```

Comment:
This is only removing the result checking, the existing test expects that the error message contains `not supported`.

I'm guessing this still produces an error, can we add a test for the error it produces now?